src/components/CommandGrid.tsx

```diff
@@ -0,0 +1,264 @@
+      'cursorForward',
+      'cursorNext',
+      'cursorPrev',
+      'jumpBack',
+      'jumpForward',
+      'moveCursorBackward',
+      'moveCursorForward',
+      'commandPalette',
+      'home',
+      'search',
+      'selectAll',
+    ],
+  },
+  {
+    title: 'Creating thoughts',
+    commands: [
+      'newThought',
+      'newThoughtAbove',
+      'newSubthought',
+      'newSubthoughtTop',
+      'newUncle',
+      'newGrandChild',
+      'subcategorizeOne',
+      'subcategorizeAll',
+      'extractThought',
+      'generateThought',
+    ],
+  },
+  {
+    title: 'Deleting thoughts',
+    commands: ['delete', 'archive', 'collapseContext', 'clearThought'],
+  },
+  {
+    title: 'Moving thoughts',
+    commands: ['indent', 'outdent', 'bumpThoughtDown', 'moveThoughtDown', 'moveThoughtUp'],
+  },
+  {
+    title: 'Editing thoughts',
+    commands: ['join', 'splitSentences', 'bold', 'italic', 'strikethrough', 'underline', 'code', 'copyCursor'],
+  },
+  {
+    title: 'Oops',
+    commands: ['undo', 'redo'],
+  },
+  {
+    title: 'Special Views',
+    commands: [
+      'note',
+      'swapNote',
+      'toggleContextView',
+      'proseView',
+      'toggleTableView',
+      'toggleSort',
+      'heading0',
+      'heading1',
+      'heading2',
+      'heading3',
+      'heading4',
+      'heading5',
+    ],
+  },
+  {
+    title: 'Visibility',
+    commands: ['pin', 'pinAll', 'toggleDone', 'toggleHiddenThoughts'],
+  },
+  {
+    title: 'Settings',
+    commands: ['customizeToolbar'],
+  },
+  {
+    title: 'Help',
+    commands: ['help', 'openGestureCheatsheet'],
+  },
+  {
+    title: 'Cancel',
+    commands: ['cancel'],
+  },
+]
+
+const commandsGroupedMap = keyValueBy(
+  groups.flatMap(group => group.commands),
+  true,
+)
+const commandsUngrouped = globalCommands.filter(
+  command => !commandsGroupedMap[command.id] && !command.hideFromHelp && (isTouch ? command.gesture : command.keyboard),
+)
+
+if (commandsUngrouped.length > 0) {
+  throw new Error(
+    `CommandTable groups are missing command(s). Please add ${conjunction(
+      commandsUngrouped.map(command => command.id),
+    )} to the appropriate group, or add hideFromHelp: true to the Command.`,
+  )
+}
+
+/** Search bar for filtering commands. */
+const SearchCommands: FC<{ onInput?: (value: string) => void }> = ({ onInput }) => {
+  const isLightTheme = useSelector(state => theme(state) === 'Light')
+
+  return (
+    <div id='search' className={css({ flexGrow: 1, border: 'solid 1px {colors.gray50}', borderRadius: '8px' })}>
+      <div className={css({ position: 'relative' })}>
+        <div
+          className={css({
+            position: 'absolute',
+            left: '10px',
+            top: '50%',
+            transform: 'translateY(-50%)',
+            display: 'flex',
+            alignItems: 'center',
+            pointerEvents: 'none',
+          })}
+        >
+          <SearchIcon size={16} fill={isLightTheme ? '#666' : '#999'} />
```

Comment:
You can define colors in https://github.com/cybersemics/em/blob/37f3e98916904810830c18b92436cad2016fac47/src/colors.config.ts. PandaCSS will automatically render the correct one for the theme.